crates/wallet/tests/psbt.rs

```diff
@@ -171,10 +171,10 @@ fn test_psbt_multiple_internalkey_signers() {
     let mut builder = wallet.build_tx();
     builder.add_recipient(send_to.script_pubkey(), 10_000);
     let mut psbt = builder.finish().unwrap();
-    // Adds a signer for the wrong internal key, bdk should not use this key to sign
+    // Adds a signer for the wrong internal key, bdk wallet should not use this key to sign
```

Comment:
```suggestion
    // Adds a signer for the wrong internal key, bdk_wallet should not use this key to sign
```
You are referring to the crate right?
Or if you are referring to the module:
```suggestion
    // Adds a signer for the wrong internal key, bdk's wallet should not use this key to sign
```

Reply:
Fixed.